### PR TITLE
DOC-2523: create separate 2013 - 2014 release notes

### PR DESCRIPTION
**Pull request by catong**
## [DOC-2523](https://openedx.atlassian.net/browse/DOC-2523)

This PR removes entries for 2013 and 2014 from the current release notes publication and creates a separate publication for RN for these years. It also updates the current release notes to link to this separate publication for older RN.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer
finishes and gives :+1:. 
- [x] Doc team review (sanity check): @pdesjardins @lamagnifica @srpearce 

FYI: Tag anyone else who might be interested in this PR here.
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### HTML Version:

http://edx.readthedocs.org/projects/edx-2013-2014-release-notes/en/latest/
- [x] Build an RTD draft for your branch and add a link here
### Post-review
- [x] Add description to release notes task as a comment
- [x] Squash commits

### CODE REVIEW

**Comment by edx-webhook** [comment]
## [DOC-XXXX](https://openedx.atlassian.net/browse/DOC-XXXX)

Add a description of your changes.


### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give
reviewers guidance on turnaround time.


### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer
finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check/copy edit/dev edit): 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.


### Testing

- [ ] Ran ./run_tests.sh without warnings or errors


### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

 
### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here


### Post-review

- [ ] Add description to release notes task as a comment
- [ ] Squash commits

File: en_us/release_notes_2014/source/01-07-2014.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ edX Studio
 
 * The **Course Outline** page is updated to include several design improvements. The new Course Outline appears as in the following example:
 
-  .. image::  images/course_outline.png
+  .. image::  Images/course_outline.png
     :alt: The Course Outline
 
   To see the changes, view your course in Studio or see Organizaing Your Course Content.
```
Comment (pdesjardins) [inline en_us/release_notes_2014/source/01-07-2014.rst]: Organizaing > Organizing. Also should this be a cross reference?

File: en_us/release_notes_2014/source/03-11-2014.rst [hunk 2]
```diff
@@ -9,7 +9,7 @@ edX Studio
 
 * When you edited an HTML component in a Chrome browser, a problem sometimes caused text that you edited in Visual view to not be updated in HTML view. This problem is resolved. (STUD-1336)
 
-* A problem that caused images to not appear in Visual view in the HTML component editor has been resolved. (STUD-1357)
+* A problem that caused Images to not appear in Visual view in the HTML component editor has been resolved. (STUD-1357)
```
Comment (pdesjardins) [inline en_us/release_notes_2014/source/03-11-2014.rst]: Looks like this was capitalized along with the file path. I think it should be lower case.

Comment (catong) [inline en_us/release_notes_2014/source/03-11-2014.rst]: Oops, yes

File: en_us/release_notes_2014/source/01-07-2014.rst (hunk 1 above)
Comment (catong) [inline en_us/release_notes_2014/source/01-07-2014.rst]: Part of the effort for this story would have been to remove all cross-reference links. However, this had already been done in some earlier effort while leaving the older entries as part of the same guide.

File: en_us/release_notes_2014/source/Images/insights_rel_notes.rtf [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1265\cocoasubrtf210
```
Comment (pdesjardins) [inline en_us/release_notes_2014/source/Images/insights_rel_notes.rtf:1]: Why is this document in rich text format? I don't really need to know, just wanted to make sure it's supposed to be that way.

Comment (catong) [inline en_us/release_notes_2014/source/Images/insights_rel_notes.rtf:1]: The same file is in the Images folder in the master branch. I'm not sure it's actually referenced in this guide. I did not check usage for all Image files, when I created the new structure.

**Comment by pdesjardins** [comment]
Looks good to me. I added some minor comments. :+1: 
Maybe add a link to the current release notes on the first page?

**Comment by catong** [comment]
Good idea re linking back to the current release notes. I'll make that change. Thanks!

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/877

**Pull request merged**